commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/model/SummaryItem.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.frontend.model;
+
+import com.liferay.portal.kernel.util.Validator;
+
+/**
+ * @author Fabio Diego Mastrorilli
+ */
+public class SummaryItem {
```

Comment:
Summary is a very generic name, can we be more specific?

Reply:
Yes why not, ideas? I reflected on java the name I used for the FE component :/